src/CP/valueselection/learning/rewards/generalreward.jl

```diff
@@ -69,9 +70,9 @@ function set_reward!(::Type{EndingPhase}, lh::LearnedHeuristic{SR, GeneralReward
 }
     # The rewards given in the EndingPhase ensure that any found feasible solution always gets a higher reward than any infeasible solution
     if symbol == :FoundSolution
-        lh.reward.value = 0
+        lh.reward.value += 0
     else
-        lh.reward.value = -lh.reward.gamma
+        lh.reward.value += -lh.reward.gamma
```

Comment:
A few months ago, we did the same mistake. Good job finding this error, does this fix is related to the unexpected reward value we had for `latin` ?

Reply:
Both of the fixes in this PR help to fix the reward values in latin. I just have to look at the testsets that it broke and we will be able to merge it into `master`.